packages/core/test/tree/treeTests.tsx

```diff
@@ -109,6 +110,14 @@ describe("<Tree>", () => {
         TestUtils.Simulate.click(document.query(`.c3 > .${Classes.TREE_NODE_CONTENT} .${Classes.TREE_NODE_CARET}`));
         assert.isTrue(onNodeCollapse.calledOnce);
         assert.deepEqual(onNodeCollapse.args[0][1], [3]);
+
+        assert.isTrue(onNodeContextMenu.notCalled);
+        // TestUtils.Simulate.contextMenu is a function, just not included in the typings
+        // nonetheless, the below line causes React to throw an error for some reason
+        // (TestUtils.Simulate as any).contextMenu(document.query(`.c0 > .${Classes.TREE_NODE_CONTENT}`));
+
+        // assert.isTrue(onNodeContextMenu.calledOnce);
+        // assert.deepEqual(onNodeContextMenu.args[0][1], [0]);
```

Comment:
This was just a simple test I wanted to run, but I couldn't get it to work. I think it was just React doing something weird because of the test environment, but unsure. I just left it in in case somebody else wanted to see if they could fix it